pystac_client/item_search.py

```diff
@@ -217,7 +217,7 @@ def __init__(
         url: str,
         *,
         method: Optional[str] = "POST",
-        max_items: Optional[int] = DEFAULT_LIMIT_AND_MAX_ITEMS,
```

Comment:
This `DEFAULT_LIMIT_AND_MAX_ITEMS` is now a misnomer, since it only applies to limit, but it's technically in the public API so I didn't rename it.

Reply:
(just resolving because of that status check)